src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -92,7 +92,7 @@ export class IFrameWebview extends BaseWebview<HTMLIFrameElement> implements Web
 		const element = document.createElement('iframe');
 		element.className = `webview ${options.customClasses || ''}`;
 		element.sandbox.add('allow-scripts', 'allow-same-origin', 'allow-forms', 'allow-pointer-lock', 'allow-downloads');
-		element.setAttribute('allow', 'clipboard-read; clipboard-write;');
+		element.setAttribute('allow', 'clipboard-read; clipboard-write; usb;');
```

Comment:
I think this should be an opt-in feature, similar to `enableScripts`

Reply:
> I think this should be an opt-in feature, similar to enableScripts

Thanks @mjbvz, I actually agree and implemented it this way initially.
However, I had issues and may need a little guidance/support.

I found that the `createElement()` function didn't contain the `contentOptions` when it is first called (e.g. the webview is displayed on startup), but only after the webview is hidden/shown. Updating the `allow` entries after the iframe is created (e.g. by overriding the `contentOptions` setter) had no effect on functionality.

I also looked into mimicking the functionality in the [pre main.js](https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/webview/browser/pre/main.js#L526) as I assume this is where the `active-frame` iframe is managed but was unable to see any change in behaviour when modifying this file. Is there a cache or something requiring invalidation if this file is modified in development?

Reply:
OK, I think I found fixed my issue. There's a remote server used to bring up the iframe and this needs to be changed to run locally by setting the `webviewEndpoint` config entry, e.g.:

```javascript
const webConfigJSON = {
    ...
    webviewEndpoint: 'http://{{uuid}}.localhost:8081/static/out/vs/workbench/contrib/webview/browser/pre'
};
```

This will then reflect any local changes to the `active-frame` webview iframe.